New Getting started guide (#1429)
getsentry/sentry-docs: src/collections/_documentation/guides/getting-started/index.md

![Add Members]({% asset guides/getting-started/04.png @path %})

You can also invite multiple new (external) members to join your organization and team by clicking on `Add Member > Invite Member`

![Invite Members]({% asset guides/getting-started/05.png @path %})

For more information see: [Organization Management](https://docs.sentry.io/accounts/membership/)

## 3. Configure Integrations

Sentry integrates seamlessly into your existing workflows by providing out of the box integrations with your favorite apps and services. To enable and configure integrations navigate to `Organization Settings > Integrations`.

### 3.1. Alert Notifications

By default, Sentry will notify you about errors in your apps via **email**. To receive real-time notifications about your errors where you work, enable Sentry integrations with: [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack), [PagerDuty](https://www.pagerduty.com/docs/guides/sentry-integration-guide/).

> ![PagerDuty Integration]({% asset guides/getting-started/slack_sentry_e2e_integration4.gif @path %})
>
> _Enabling alert notifications through the __Slack__ integration._

> Sentry supports additional solutions on a project level. After creating a project - **Hipchat**, **Twilio**, **OpsGenie**, and other solutions can be configured and used to channel notifications through alert rules. For more information check out:

Reviewer: ```suggestion
Sentry supports additional solutions on a project level. After creating a project, **Hipchat**, **Twilio**, **OpsGenie**, and other solutions can be configured and used to channel notifications through alert rules. For more information, check out:
```

- [Per-Project Integrations: Issue Notifications](https://docs.sentry.io/workflow/integrations/legacy-integrations/#issue-notifications)
- [Routing Alerts to 3rd Party Integrations](https://docs.sentry.io/guides/alert-notifications/routing-alerts/#routing-alerts-to-3rd-party-integrations)

### 3.2. Source Code Management

By enabling Sentry's integration with your Source Code Management (SCM) provider, Sentry will:

1. Identify **suspect commits** that likely introduced an error.
2. **Suggest assignees** who can best resolve an error.
3. Mark an issue as **Resolved via Commit or PR** to stop notifications about similar errors in past releases and notify you upon regression.
4. Link a Sentry issue to a new or existing issue in your integrated [Issue Tracking](#23-issue-tracking) solution

    ![Integrate SCMs]({% asset guides/getting-started/07.png @path %})

> Sentry has built-in integrations with: [GitHub](https://docs.sentry.io/workflow/integrations/global-integrations/#github), [GitHub Enterprise](https://docs.sentry.io/workflow/integrations/global-integrations/#github-enterprise), [GitLab](https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab), and [Bitbucket](https://docs.sentry.io/workflow/integrations/global-integrations/#bitbucket). If you're using a different SCM provider or don't want Sentry to connect to your repository, checkout our [docs]((https://docs.sentry.io/workflow/releases/?platform=browser#alternatively-without-a-repository-integration)) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.

Reviewer: ```suggestion
Sentry has built-in integrations with: 
- [GitHub](https://docs.sentry.io/workflow/integrations/global-integrations/#github)
- [GitHub Enterprise](https://docs.sentry.io/workflow/integrations/global-integrations/#github-enterprise)
- [GitLab](https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab) 
- [Bitbucket](https://docs.sentry.io/workflow/integrations/global-integrations/#bitbucket). 

If you're using a different SCM provider or don't want Sentry to connect to your repository, checkout our [docs]((https://docs.sentry.io/workflow/releases/?platform=browser#alternatively-without-a-repository-integration)) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
```

Reviewer: The GitHub and GitLab links can get confused with each other. Breaking them into bullet points might be easier to differentiate.

### 3.3. Issue Tracking

Enabling the 2-way integration with your Issue Tracking solution allows you to create a new issue from within the Sentry issue page, or link it to an existing one. In addition to **GitHub**, **GitLab**, and **Bitbucket** Issues that are enabled with the respective SCM integration, Sentry also integrates with: [Azure DevOps](https://docs.sentry.io/workflow/integrations/global-integrations/#azure-devops), [Clubhouse](https://docs.sentry.io/workflow/integrations/global-integrations/#clubhouse), [JIRA](https://docs.sentry.io/workflow/integrations/global-integrations/#jira), and others.

> ![JIRA 2-Way Integration]({% asset guides/getting-started/jira-2way-binding.gif @path %})
>
> 2-Way integration with JIRA issue tracking.

For a full list of all supported integrations, go to our documentation on [Integrations](https://docs.sentry.io/workflow/integrations/)

## 4. Create Projects

> To start monitoring errors in your app with Sentry you'll need to initialize the SDK with a DSN key. To obtain a key, simply create a new Sentry project by going to `Settings > Projects > + Create Project`. Give the project a name and assign the responsible [Team/s](#3-set-up-teams). Now, any error that occurs in your code (wherever it may be deployed and running) will be associated with that specific project.

Reviewer: ```suggestion
To start monitoring errors in your app with Sentry, you'll need to initialize the SDK with a DSN key. To obtain a key, create a new Sentry project by going to `Settings > Projects > + Create Project`. Give the project a name and assign the responsible [Team/s](#3-set-up-teams). Now, any error that occurs in your code (wherever it may be deployed and running) will be associated with that specific project.
```

![Create a Project]({% asset guides/getting-started/08.png @path %})

### > **What's in a Project?**

> A Project is a logical entity that connects between the errors in your application (or a part of it) to the team members assigned to that project. The **Project Settings** configuration determines

Reviewer: ```suggestion
A Project is a logical entity that connects between the errors in your application (or a part of it) to the team members assigned to that project. The **Project Settings** configuration determines:
```

- Who to notify, about what error, and how, using [Alert Rules](https://docs.sentry.io/workflow/notifications/alerts/)
- Which errors should be automatically assigned to which member or team through [Issue Owners](https://docs.sentry.io/workflow/issue-owners/)
- Custom rules to fine-tune [Event Grouping](https://docs.sentry.io/data-management/event-grouping/) into issues.

Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, developers are looking at information that is relevant to their work, and see the errors that they might need to jump on.

![My Issue Stream]({% asset guides/getting-started/09.png @path %})

### > **How Many Projects Should I Create?**

You could theoretically pour all your errors into a single project. The Sentry UI will not limit you. However, considering the above ([What's in a Project?](#-whats-in-a-project)), setting up multiple projects to reflect your application landscape with finer granularity, will go a long way with regards to visibility, actionability, and productivity.

Here are some points to consider:

> - In case your application's source code is managed in **multiple repositories** - create a **separate project for each repo**.

Reviewer: ```suggestion
- In case your application's source code is managed in **multiple repositories**, create a **separate project for each repo**.
```

> - In case your app is made up of **several micro-services** - **split them into projects** accordingly.

Reviewer: ```suggestion
- In case your app is made up of **several micro-services**, **split them into projects** accordingly.
```

> - If you have a **monolithic codebase** - set up **separate projects for the backend and frontend**.

Reviewer: ```suggestion
- If you have a **monolithic codebase**, set up **separate projects for the backend and frontend**.
```

> - Give **each language its own project** (e.g. your backend code contains NodeJS and Java components, separate those into two different projects).  

Reviewer: ```suggestion
- Give **each language its own project** (for example, your backend code contains NodeJS and Java components, separate those into two different projects).  
```

## 5. Define Environments

> Depending on your development life cycle, your applications are probably deployed and running in multiple environments -  _QA_, _Staging_, _Production_, etc. Configuring the environment in your Sentry SDK, will add the tag to every error event and will help you associate events to the environment in which they occurred. You can do that by creating a `SENTRY_ENVIRONMENT` environment variable or by explicitly defining it in the SDK configuration. For more information see our documentation on [Environments](https://docs.sentry.io/enriching-error-data/environments/?platform=browser).

Reviewer: ```suggestion
Depending on your development life cycle, your applications are probably deployed and running in multiple environments --  _QA_, _Staging_, _Production_, etc. Configuring the environment in your Sentry SDK, will add the tag to every error event and will help you associate events to the environment in which they occurred. You can do that by creating a `SENTRY_ENVIRONMENT` environment variable or by explicitly defining it in the SDK configuration. For more information, see our documentation on [Environments](https://docs.sentry.io/enriching-error-data/environments/?platform=browser).
```

### 2.3. Issue Tracking

Enabling the 2-way integration with your Issue Tracking solution allows you to create a new issue from within the Sentry issue page, or link it to an existing one. In addition to **GitHub**, **GitLab**, and **Bitbucket** Issues that are enabled with the respective SCM integration, Sentry also integrates with: [Azure DevOps](https://docs.sentry.io/workflow/integrations/global-integrations/#azure-devops), [Clubhouse](https://docs.sentry.io/workflow/integrations/global-integrations/#clubhouse), [JIRA](https://docs.sentry.io/workflow/integrations/global-integrations/#jira), and others.

> ![JIRA 2-Way Integration]({% asset guides/getting-started/jira-2way-binding.gif @path %})
>
> 2-Way integration with JIRA issue tracking.

For a full list of all supported integrations, go to our documentation on [Integrations](https://docs.sentry.io/workflow/integrations/)

> ## 3. Set-up Teams

Reviewer 2: i guess the UI onboarding just takes them to join teams once are signed in for the first time

- Who to notify, about what error, and how, using [Alert Rules](https://docs.sentry.io/workflow/notifications/alerts/)
- Which errors should be automatically assigned to which member or team through [Issue Owners](https://docs.sentry.io/workflow/issue-owners/)
- Custom rules to fine-tune [Event Grouping](https://docs.sentry.io/data-management/event-grouping/) into issues.

Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, developers are looking at information that is relevant to their work, and see the errors that they might need to jump on.

![My Issue Stream]({% asset guides/getting-started/09.png @path %})

### > **How Many Projects Should I Create?**

You could theoretically pour all your errors into a single project. The Sentry UI will not limit you. However, considering the above ([What's in a Project?](#-whats-in-a-project)), setting up multiple projects to reflect your application landscape with finer granularity, will go a long way with regards to visibility, actionability, and productivity.

Here are some points to consider:

> - If your app source code is managed in multiple repositories, create a separate project for each repo.

Reviewer 2: Can we add some formatting / bolding here to each of these entries. i.e. **several micro-service** -> **split**

Up to you (:

- Is your app made up of several micro-services? Split those into projects accordingly.
- If you have a monolithic codebase, set up separate projects for the Backend and Frontend.
- Give each language its own project. If your backend code has NodeJS and Java components, separate those into two different projects.  

## 5. Define Environments

> Depending on your development life cycle, your applications are probably deployed and running in multiple environments -  _Dev_, _Testing_, _Production_, etc. Configuring the environment in your Sentry SDK, will add the tag to every error event and will help you associate events to the environment in which they occurred. You can do that by creating a `SENTRY_ENVIRONMENT` environment variable or by explicitly defining it in the SDK configuration. For more information see our documentation on [Environments](https://docs.sentry.io/enriching-error-data/environments/?platform=browser).

Reviewer 2: Technically dev/localhost errors shouldn't be going to sentry. let's remove mentions of dev
